bundle/bundle.go

```diff
@@ -293,3 +293,10 @@ func WithAllAvailableClients() Option {
 		b.urls.PutAllAvailable()
 	}
 }
+
+// WithCustom # TODO remove it after internal bundle invoke inside cp issue-manage adjusted
```

Comment:
remove TODO here.

move TODO to dop provider with `localhost:9527.`

Reply:
done